packages/ui/src/lib/dropdownMenu/DropDownMenuItems.svelte

```diff
@@ -1,13 +1,18 @@
 <script lang="ts">
-import { onDestroy, onMount } from 'svelte';
+import { onDestroy, onMount, type Snippet } from 'svelte';
 
 let dropDownHeight: number;
 let dropDownWidth: number;
 let dropDownElement: HTMLElement;
 let sideAlign: string;
```

Comment:
Question for @feloy @deboer-tim 

For these, when I added props, I get the warning: 
```
sideAlign is updated, but is not declared with `$state(...)`. Changing its value will not correctly trigger updates
https://svelte.dev/e/non_reactive_updatesvelte(non_reactive_update)`
```

Although I see that functionality isn't effected, and it does not say if it's svelte5 migration feature, etc.

Is this safe to ignore? If not, how did you do the changes in your migration files?

Reply:
Yes, you'll need to do this too. For Svelte 5 all variables need to be marked as either state or derived values. You can find the full details in the migration guide (https://svelte.dev/docs/svelte/v5-migration-guide#Reactivity-syntax-changes) but most state like:
  `let dropDownHeight: number;`
will need to be changed to:
  `let dropDownHeight = $state(0);`
or:
  `let dropDownHeight = $state<number>();`